docs/conf/eu/2016/news/APIdays-blurb.md

```diff
@@ -0,0 +1,3 @@
+## Write the Docs EU, September 18-20, Prague, CZ
+
+To all members of the API community who care about good docs -- and who doesn't? -- Write the Docs, a conference for everyone who cares about documentation (developers, writers, designers, architects, and more) is taking place this year in Prague, CZ, on September 18-20. There will be a wide range of talks, including several specifically about API docs, from GraphQL as an API definition language to usability studies of API docs (with data!) to the importance of naming in API design. You should attend! For more information, see the [conference website](http://www.writethedocs.org/conf/eu/2016/) or [purchase tickets](https://ti.to/writethedocs/write-the-docs-eu-2016?_=). Documentarians are a friendly bunch, and they'd love to see you there!
```

Comment:
This looks good.

I'm not hugely keen on "all members of the api community... conference for everyone who cares about docs" feels a little heavy and circular. But I don't have an amazing suggestion for improvement, aside from cutting out everything before Write the Docs which I don't think is the way to go.